policy/predicate/file.go

```diff
@@ -141,6 +141,44 @@ func (pred *OnlyChangedFiles) Trigger() common.Trigger {
 	return common.TriggerCommit
 }
 
+type NoChangedFiles struct {
+	Paths       []common.Regexp `yaml:"paths"`
+	IgnorePaths []common.Regexp `yaml:"ignore"`
+}
+
+var _ Predicate = &NoChangedFiles{}
+
+func (pred *NoChangedFiles) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
+	changedFiles := ChangedFiles{
+		Paths:       pred.Paths,
+		IgnorePaths: pred.IgnorePaths,
+	}
+
+	changedFilesPredicateResult, err := changedFiles.Evaluate(ctx, prctx)
+	if err != nil {
+		return nil, err
+	}
+
+	predicateResult := common.PredicateResult{
+		Values:          changedFilesPredicateResult.Values,
+		Description:     changedFilesPredicateResult.Description,
+		Satisfied:       !changedFilesPredicateResult.Satisfied,
+		ValuePhrase:     "changed files",
+		ConditionPhrase: "cannot match",
+		ConditionsMap:   changedFilesPredicateResult.ConditionsMap,
+	}
+
+	if changedFilesPredicateResult.Satisfied {
+		predicateResult.Description = "No changed files match the excluded patterns"
+	}
```

Comment:
I think this is condition is inverted. When `changedFilesPredicateResult.Satisfied` is true, the `Description` field contains the file that matched, which is the correct explanation for why this rule is _not_ satisfied. When it is false, we can change the description to "No changed files match the specified patterns"

(I think "excluded" gives the wrong idea, so "specified" is a bit more generic than the original "required")